libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -1,101 +1,89 @@
-    project ("transportsecure sockets unit test")
-    cmake_minimum_required (VERSION 3.13)
+project ("transportsecure sockets unit test")
+cmake_minimum_required (VERSION 3.13)
 
 # ====================  Define your project name (edit) ========================
-    set(project_name "transport_secure_sockets")
+set(project_name "transport_secure_sockets")
 
 # =====================  Create your mock here  (edit)  ========================
 
 # list the files to mock here
-    list(APPEND mock_list
-                "${kernel_dir}/include/task.h"
-                "${kernel_dir}/include/portable.h"
-                "${common_dir}/include/iot_logging_task.h"
-                "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
-            )
+list(APPEND mock_list
+            "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
+        )
 # list the directories your mocks need
-    list(APPEND mock_include_list
-                "${CMAKE_CURRENT_LIST_DIR}/include"
-            )
+list(APPEND mock_include_list
+            "${CMAKE_CURRENT_LIST_DIR}/include"
+        )
 #list the definitions of your mocks to control what to be included
-    list(APPEND mock_define_list
-                LWIP_COMPAT_SOCKETS=1
-                portHAS_STACK_OVERFLOW_CHECKING=1
-                portUSING_MPU_WRAPPERS=1
-                MPU_WRAPPERS_INCLUDED_FROM_API_FILE
-           )
+list(APPEND mock_define_list
+            LWIP_COMPAT_SOCKETS=1
+            portHAS_STACK_OVERFLOW_CHECKING=1
+            portUSING_MPU_WRAPPERS=1
+            MPU_WRAPPERS_INCLUDED_FROM_API_FILE
```

Comment:
These defines may not be required for `transport_secure_sockets` unit test

Reply:
removed in f3c02d7